config-base.js

```diff
@@ -107,19 +112,32 @@ module.exports = {
     'unicorn/catch-error-name': 'off',
     'unicorn/consistent-function-scoping': 'off',
     'unicorn/no-abusive-eslint-disable': 'off',
+    'unicorn/no-array-callback-reference': 'off',
+    'unicorn/no-array-for-each': 'off',
+    'unicorn/no-await-expression-member': 'off',
     'unicorn/no-fn-reference-in-iterator': 'off',
+    'unicorn/no-lonely-if': 'off',
     'unicorn/no-nested-ternary': 'warn',
     'unicorn/no-null': 'off',
     'unicorn/no-process-exit': 'off',
     'unicorn/no-reduce': 'warn',
+    'unicorn/numeric-separators-style': 'off',
+    'unicorn/prefer-module': 'off',
+    'unicorn/prefer-node-protocol': 'off',
```

Comment:
I want to turn this on but I don't think we can yet because of minimum Node version and TypeScript